src/Akka.Persistence.Sql/Journal/Dao/BaseByteArrayJournalDao.cs

```diff
@@ -29,38 +29,6 @@
 
 namespace Akka.Persistence.Sql.Journal.Dao
 {
-    public static class SequentialUuidGenerator
```

Comment:
Remove the old Uuid generator, we'll be using the default randomized Guid instead.